.github/workflows/go_test_integration.yml

```diff
@@ -57,4 +57,4 @@ jobs:
       - name: Test Go code
         run: |
           export KO_DOCKER_REPO="kind.local"
-          go test -count=1 -v ./test/integration -tags=integration -timeout 60m
+          go test -count=1 -v ./test/integration -tags=integration
```

Comment:
Be careful with this change. This can't be merged until we make sure the integration tests run under 10 minutes each one

Reply:
How many executions do you need to be comfortable merging this?

Reply:
I wasn't thinking on an amount of executions but rather having addressed #776 before